### PR TITLE
docs: Add references to "ci" and "packaging" repos

### LINKED ISSUE #6: Update README for new repos
Update `README.md` for:

- https://github.com/kata-containers/ci
- https://github.com/kata-containers/packaging

### PR DESCRIPTION
**Pull request by jodh-intel**
Add brief details of the new "ci" and "packaging" repositories.

Fixes #6.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

**Comment by jodh-intel** [comment]
Hi @klynnrif, @rcaballeromx - could you take a look please?

File: README.md [hunk 1]
```diff
@@ -48,6 +50,11 @@ issues](https://github.com/kata-containers/kata-containers/issues/new):
 
 ## Kata Containers repositories
 
+### CI
+
+The [CI](https://github.com/kata-containers/ci) repository stores Continuous
+Integration (CI) configuration.
```
Comment (grahamwhaley) [inline README.md]: maybe expand 'configuration' to 'configurations' or 'system configuration information' or similar

File: README.md [hunk 2]
```diff
@@ -97,16 +104,21 @@ create virtual machines for hosting the containers.
...
 ### Documentation
 
-The [`documentation`](https://github.com/kata-containers/documentation)
+The [documentation](https://github.com/kata-containers/documentation)
 repository hosts documentation common to all code components.
 
+### Packaging
+
+The [packaging](https://github.com/kata-containers/packaging)
+repository is used for creating packages for system components.
```
Comment (grahamwhaley) [inline README.md]: optionally add something like 'including rootfs images and linux kernels' maybe.

File: README.md (hunk 1 above)
Comment (jodh-intel) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (jodh-intel) [reply README.md]: Done.

File: README.md [hunk 3]
```diff
@@ -97,22 +104,29 @@ create virtual machines for hosting the containers.
 
 ##### Kernel
 
-The [`linux`](https://github.com/kata-containers/linux) kernel is used by the hypervisor to boot the guest image.
+The [Linux\*](https://github.com/kata-containers/linux) kernel is used by the hypervisor to boot the guest image.
```
Comment (rcaballeromx) [inline README.md]: Passive voice. Consider:
The hypervisor uses the  [Linux\* kernel](https://github.com/kata-containers/linux) to boot the guest image.

File: README.md [hunk 4]
```diff
@@ -48,6 +50,11 @@ issues](https://github.com/kata-containers/kata-containers/issues/new):
 
 ## Kata Containers repositories
 
+### CI
+
+The [CI](https://github.com/kata-containers/ci) repository stores Continuous
```
Comment (rcaballeromx) [inline README.md]: Missing article. Consider adding:
... stores the Continuous ...

File: README.md [hunk 5]
```diff
@@ -97,22 +104,29 @@ create virtual machines for hosting the containers.
...
 
 ### Documentation
 
-The [`documentation`](https://github.com/kata-containers/documentation)
+The [documentation](https://github.com/kata-containers/documentation)
 repository hosts documentation common to all code components.
 
+### Packaging
+
+The [packaging](https://github.com/kata-containers/packaging) repository is
```
Comment (rcaballeromx) [inline README.md]: Passive voice. Consider:
We use the [packaging repository](https://github.com/kata-containers/packaging) to create packages for the [system components](#kata-containers-developed-components) including [rootfs](#os-builder) and the [kernel](#kernel) images.

**Pull request merged**